Assets/Scripts/Game.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+public class Game : IDisposable
```

Comment:
IDisposable won't work 
DI Framework has its own set of interfaces: IStartable, IDestroyable, etc.